### ISSUE TITLE
aws-analytics-pinpoint:2.1.0 breaking merged AndroidManifest.xml

### ISSUE THREAD

**Issue by flochaz** [aws-amplify/amplify-android#2249]
### Before opening, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-android/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-android/discussions).

### Language and Async Model

Kotlin - Coroutines

### Amplify Categories

Analytics

### Gradle script dependencies

<details>

```groovy
plugins {
    id 'com.android.application'
    id 'org.jetbrains.kotlin.android'
}

android {
    namespace 'com.example.travelfootprintcalculator'
    compileSdk 33

    defaultConfig {
        applicationId "com.example.travelfootprintcalculator"
        minSdk 24
        targetSdk 33
        versionCode 1
        versionName "1.0"

        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
    }

    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
        }
    }
    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }
    kotlinOptions {
        jvmTarget = '1.8'
    }
}

dependencies {

    implementation 'androidx.core:core-ktx:1.7.0'
    implementation 'androidx.appcompat:appcompat:1.4.1'
    implementation 'com.google.android.material:material:1.5.0'
    implementation 'androidx.constraintlayout:constraintlayout:2.1.3'
    implementation 'com.amplifyframework:aws-api:2.1.0'
    implementation 'com.amplifyframework:aws-datastore:2.1.0'
    implementation 'com.amplifyframework:aws-auth-cognito:2.1.0'
    implementation 'com.amplifyframework:aws-analytics-pinpoint:2.1.0'
    implementation 'com.amplifyframework:aws-geo-location:2.1.0'
    implementation 'androidx.lifecycle:lifecycle-viewmodel:2.5.1'
    implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.5.1'
    testImplementation 'junit:junit:4.13.2'
    androidTestImplementation 'androidx.test.ext:junit:1.1.3'
    androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'

}

```

</details>

### Environment information

<details>

```
❯   ./gradlew --version


Welcome to Gradle 7.5!

Here are the highlights of this release:
 - Support for Java 18
 - Support for building with Groovy 4
 - Much more responsive continuous builds
 - Improved diagnostics for dependency resolution

For more details see https://docs.gradle.org/7.5/release-notes.html


------------------------------------------------------------
Gradle 7.5
------------------------------------------------------------

Build time:   2022-07-14 12:48:15 UTC
Revision:     c7db7b958189ad2b0c1472b6fe663e6d654a5103

Kotlin:       1.6.21
Groovy:       3.0.10
Ant:          Apache Ant(TM) version 1.10.11 compiled on July 10 2021
JVM:          11.0.17 (Amazon.com Inc. 11.0.17+8-LTS)
OS:           Mac OS X 12.6.2 x86_64



```

</details>

### Please include any relevant guides or documentation you're referencing

https://docs.amplify.aws/lib/analytics/getting-started/q/platform/android/

### Describe the bug

When adding     `implementation 'com.amplifyframework:aws-analytics-pinpoint:2.1.0'` to my dependencies on a brand new Android App created with Android Studio, I am getting a merge issue on AndroidManifest.xml.

`[androidx.test.core.app.InstrumentationActivityInvoker$BootstrapActivity](android:exported needs to be explicitly specified for element)` 

### Reproduction steps (if applicable)

1. Create Android app with android studio (Electric Eel | 2022.1.1)
2. Add `com.amplifyframework:aws-analytics-pinpoint:2.1.0` to your deps
3. build

### Log output

<details>

```
> Task :app:processDebugMainManifest FAILED
app/src/main/AndroidManifest.xml:27:9-33:20 Error:
	android:exported needs to be explicitly specified for element <activity#androidx.test.core.app.InstrumentationActivityInvoker$BootstrapActivity>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.
app/src/main/AndroidManifest.xml:34:9-40:20 Error:
	android:exported needs to be explicitly specified for element <activity#androidx.test.core.app.InstrumentationActivityInvoker$EmptyActivity>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.
app/src/main/AndroidManifest.xml:41:9-47:20 Error:
	android:exported needs to be explicitly specified for element <activity#androidx.test.core.app.InstrumentationActivityInvoker$EmptyFloatingActivity>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.
```

</details>

### Additional information and screenshots

Workaround:

```groovy
implementation('com.amplifyframework:aws-analytics-pinpoint:2.1.0'){
        exclude group: "androidx.test.ext", module: "junit"
        exclude group: "androidx.test", module: "core"
}
```

**Comment by mattcreaser** [comment]
Thanks for pointing this out @flochaz. We'll make a fix here.

**Comment by mattcreaser** [comment]
Hi @flochaz this should be fixed in 2.2.0. Thanks for the report!
